knexfile.js

```diff
@@ -0,0 +1,49 @@
+// Update with your config settings.
+const pgConnection = process.env.DATABASE_URL;
+
+module.exports = {
```

Comment:
Good Job keeping the Environment Variables  and db connection strings out of the public repo. 👍